sematic/plugins/kuberay_wrapper/standard.py

```diff
@@ -190,6 +219,21 @@ def create_cluster_manifest(  # type: ignore
         manifest = deepcopy(cls._manifest_template)
         manifest["metadata"]["name"] = cluster_name
         manifest["spec"]["rayVersion"] = cluster_config.ray_version
+        manifest["spec"][
+            "enableInTreeAutoscaling"
+        ] = cluster_config.requires_autoscale()
+
+        autoscaler_config = AutoscalerConfig(
+            cpu=0.5,
+            memory_gb=1,
+        )
```

Comment:
[Nit] Does this not fit on one line? It would be more compact and readable.

Reply:
This is for `black` to decide, we do not discuss formatting in PRs. 🙃

Reply:
That's what I mean by "fit" - if black doesn't complain if it is on one line.

Reply:
`black` formatted it to this